src/pks/transport/transport_ats_pk.cc

```diff
@@ -1016,8 +1044,13 @@ Transport_ATS::AdvanceAdvectionSources_RK1_(double t_old,
     *S_->GetW<CompositeVector>(conserve_qty_key_, tag_next_, name_)
     .ViewComponent("cell", false);
 
-  // -- M <-- C0 * WC0
-  // -- mol H2O * (mol C / mol H20) --> mol C, the conserved component quantity
+  // populating solute mass flux (unit: molC)
+  Epetra_MultiVector& mass_flux_advection = 
+    *S_->GetW<CompositeVector>(mass_flux_advection_key_, tag_next_, name_)
+    .ViewComponent("face", false);  
+
+  // Mass <-- Concentration * water content
```

Comment:
// Mass_0 <-- Mole Fraction_0 * Water Content_0

(or otherwise note that this is being done at the "old" time to compute the initial mass)